pinot-core/src/main/java/org/apache/pinot/core/data/function/FunctionRegistry.java

```diff
@@ -18,36 +18,26 @@
  */
 package org.apache.pinot.core.data.function;
 
-import com.google.common.base.Preconditions;
 import java.lang.reflect.Method;
-import java.lang.reflect.Modifier;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 
 /**
  * Registry for inbuilt Pinot functions
  */
 public class FunctionRegistry {
```

Comment:
Thoughts on changing the name of this to `DefaultFunctionRegistry` now that it only applies to the DefaultFunctionEvaluator?